test/unit/mocks.js

```diff
@@ -1,12 +1,79 @@
 define(["ObjectMakr"], function (ObjectMakrModule) {
     var ObjectMakr = ObjectMakrModule.ObjectMakr;
+    var expect = require("chai").expect;
 
     var mocks = {
         /**
-         * 
+         * @returns instance of ObjectMakr.
          */
         mockObjectMakr: function (settings) {
-            return new ObjectMakr(unitsize)
+            return new ObjectMakr(settings || {
+                "inheritance": {
+                    "Animal": {
+                        "Bird":{
+                            "Penguin": {}
+                        },
+                        "Mammal": {
+                            "Dog": {}
+                        }
+                    }
+                },
+                "properties": {
+                    "Animal": {
+                        "name": "",
+                        "weight": 0,
+                        "age": 0
+                    },
+                    "Penguin": {
+                        "onMake": function(penguin){
+                            penguin.name = penguin.name + " the Penguin"
+                        }
+                    },
+                    "Dog": {
+                        "onMake": function(dog){
+                            dog.name = dog.name + " the Fluffy Dog"
+                        }
+                    }
+                },
+                "doPropertiesFull": true,
+                "indexMap": ["name", "weight", "age"],
+                "onMake": "onMake"
+            })
+        },       
+        /**
+         * Specific class name for tests to use.
+         */
+        mockClassName: "Penguin",
+        /**
+         * @returns an object with characteristic properties
```

Comment:
I changed the line to read 'An object with properties of different types.'
Should i tack on 'for a thing' onto the end of that to make it more clear? 'a thing' is still vague but is more descriptive I think.